sdk/java/dagger-java-sdk/src/main/java/io/dagger/client/Convert.java

```diff
@@ -0,0 +1,57 @@
+package io.dagger.client;
+
+import com.google.gson.Gson;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.concurrent.ExecutionException;
+
+public interface Convert {
```

Comment:
Maybe replace `interface` by a `final class`. It is a good practice for utility classes and it prevents Convert to be implemented